src/utils/address/addAddressToDb.ts

```diff
@@ -0,0 +1,22 @@
+import address from '../../models/address';
+import returnObj from '../returnObj';
+import { addressType } from '../interfaces';
+
+const addAddressToDb = async (addressUser: string) => {
+  const addressDetail: addressType = {
+    address: addressUser,
+  };
+
+  try {
+    const addressDb = new address(addressDetail);
+    await addressDb.save();
+    return returnObj(true, 'add token to database', addressDb);
+  } catch (e) {
+    if (e instanceof Error) {
+      console.log(e.message);
+      return returnObj(false, e.message, null);
+    }
+  }
+};
```

Comment:
rename to addAlreadyMintedToDB

rename addressUser to user

move addressDetail to new Address()

rename addressDb to newAlreadyMinted

remove the console

change message to 'User added to AlreadyMinted successfully'.

remove the if in the catch if possible